cli/tauri-bundler/src/error.rs

```diff
@@ -54,6 +53,14 @@ pub enum Error {
   ShellScriptError(String),
   #[error("`{0}`")]
   GenericError(String),
+  #[error("Unable to create updater package")]
+  UpdateBundler,
+  #[error("Decode Error: `{0}`")]
```

Comment:
```suggestion
  #[error(transparent)]
```
You could use `#[error(transparent)]` to forward the message from other errors.